# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../yaboot/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

diff -ur yaboot-1.3.11-orig/lib/strstr.c yaboot-1.3.11/lib/strstr.c
--- yaboot-1.3.11-orig/lib/strstr.c	2001-09-20 15:15:31.000000000 +0200
+++ yaboot-1.3.11/lib/strstr.c	2004-05-04 11:23:49.000000000 +0200
@@ -76,7 +76,9 @@
 	      a = *++haystack;
 	      if (a == '\0')
 		goto ret0;
-shloop:	    }
+shloop:
+	  ((void)0);
+          }
           while (a != b);
 
 jin:	  a = *++haystack;
